IoTuring/Entity/Deployments/Wifi/Wifi.py

```diff
@@ -0,0 +1,410 @@
+                    "Señal": r"Señal:\s+(\d+%?)",
+                    "Perfil": r"Perfil:\s+(.+)"
+                }
+            },
+            OsD.LINUX: {
+                "en" : { 
+                    "ESSID": r'ESSID:"([^"]*)"',
+                    "Mode": r"Mode:([^\s]+)",
+                    "Frequency": r"Frequency:([\d.]+) (GHz|Mhz)",
+                    "Access_Point": r"Access Point: ([\w:]+)",
+                    "Bit_Rate": r"Bit Rate=([\d.]+) (\w+/s)",
+                    "Tx_Power": r"Tx-Power=(-?\d+) (\w+)",
+                    "Link_Quality": r"Link Quality=(\d+/\d+)",
+                    "Signal_Level": r"Signal level=(-?\d+) (\w+)",
+                    "Rx_invalid_nwid": r"Rx invalid nwid:(\d+)",
+                    "Rx_invalid_crypt": r"Rx invalid crypt:(\d+)",
+                    "Rx_invalid_frag": r"Rx invalid frag:(\d+)",
+                    "Tx_excessive_retries": r"Tx excessive retries:(\d+)",
+                    "Invalid_misc": r"Invalid misc:(\d+)",
+                    "Missed_beacon": r"Missed beacon:(\d+)",
+                },
+            },
+            OsD.MACOS: { # no language differentiation in macos: always english
+                "en": {
+                    "agrCtlRSSI": r"[^\n][\s]*agrCtlRSSI:\s+(-?\d+)\n",
+                    "agrExtRSSI": r"[^\n][\s]*agrExtRSSI:\s+(-?\d+)\n",
+                    "agrCtlNoise": r"[^\n][\s]*agrCtlNoise:\s+(-?\d+)\n",
+                    "agrExtNoise": r"[^\n][\s]*agrExtNoise:\s+(-?\d+)\n",
+                    "state": r"[^\n][\s]*state:\s+(\w+)\n",
+                    "op mode": r"[^\n][\s]*op mode:\s+(\w+)\n",
+                    "lastTxRate": r"[^\n][\s]*lastTxRate:\s+(\d+)\n",
+                    "maxRate": r"[^\n][\s]*maxRate:\s+(\d+)\n",
+                    "lastAssocStatus": r"[^\n][\s]*lastAssocStatus:\s+(\d+)\n",
+                    "802.11 auth": r"[^\n][\s]*802\.11 auth:\s+(\w+)\n",
+                    "link auth": r"[^\n][\s]*link auth:\s+(\w+-\w+)\n",
+                    "BSSID": r"[^\n][\s]*BSSID:\s+([\w:]+)\n",
+                    "SSID": r"\n[\s]*SSID:\s+([\w\s]+)\n",
+                    "MCS": r"[^\n][\s]*MCS:\s+(\d+)\n",
+                    "channel": r"[^\n][\s]*channel:\s+([\d,]+)\n",
+                }
+            },
+        }
+        self.platform = OsD.GetOs()
+        self.locale_str, _ = locale.getlocale()
+        self.language: str = self.locale_str.split("_")[0]
+
+        # In macos trick the language to be english since the output of airport is always in english
+        if OsD.IsMacos():
+            self.language = "en"
+
+        if OsD.IsWindows():
+            self.keySignalStrength = KEY_SIGNAL_STRENGTH_PERCENT
+            self.valueFormatterOptionsSignalStrength =  VALUEFORMATTEROPTIONS_PERCENTAGE
+        else:
+            self.keySignalStrength = KEY_SIGNAL_STRENGTH_DBM
+            self.valueFormatterOptionsSignalStrength = VALUEFORMATTEROPTIONS_DBM
```

Comment:
@richibrics
Shouldnt this already do the trick?
I'll add the conversion from percent to dbm
Maybe we let the user configure if he wants percent or dbm?

Reply:
![image](https://github.com/richibrics/IoTuring/assets/96825051/51d98409-1396-44c5-b1c5-7fc437012aea)
What about this? 

I'd be fine with defaulting to dBm, since the signal in percent will be sent as an extra attribute in windows anyways